iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -375,7 +376,7 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
             try
             {
                 await EnableTwinPatchAsync(cancellationToken).ConfigureAwait(false);
-                Twin twin = await RoundTripTwinMessage(AmqpTwinMessageType.Patch, reportedProperties, cancellationToken).ConfigureAwait(false);
+                await RoundTripTwinMessage(AmqpTwinMessageType.Patch, reportedProperties, cancellationToken).ConfigureAwait(false);
```

Comment:
are we intentionally ignoring the returned twin?

Reply:
Yes no changes really. This is same as before, we were not doing anything with returned twin